scylladb/sphinx-scylladb-theme
# Fix secondary sidebar scroll

Fixes a bug introduced in https://github.com/scylladb/sphinx-scylladb-theme/pull/1051 that affects the secondary sidebar highlighting.

How to reproduce:

Open https://sphinx-theme.scylladb.com/stable/getting-started/quickstart.html#step-2-add-the-page-to-the-toc-tree. The second step should be highlighted on the secondary sidebar, but the first one is instead.

## How to test

1. Clone this PR.
2. Build the docs.
3. Open http://127.0.0.1:5500/getting-started/quickstart/#step-2-add-the-page-to-the-toc-tree
4. Confirm that the secondary sidebar correctly highlights Step 2
